Matrix.py

```diff
@@ -0,0 +1,168 @@
+"""matrices calculator class."""
+
+
+class Matrix:
+    """A matrices calculator for basic math actions
+        Attributes:
+            data(tuple of tuples): a Matrix- each line is a tuple in the major tuple """
+
+    def __init__(self, data):
+        if type(data) is not tuple:
+            raise ValueError("Invalid matrix")
+        for line in data:
+            if len(line) != len(data) or type(line) is not tuple:
+                raise ValueError("Invalid matrix")
+        self.data = data
+
+    def __iter__(self):
+        return iter(self.data)
+
+    def __getitem__(self, item):
+        return self.data[item]
+
+    def __repr__(self):
+        """prints the matrix in a format that can be used for build another Matrix"""
+        return f"{self.__class__.__name__}({self.data})"
+
+    def __str__(self):
+        matrix_in_lines = ""
+        for i in range(len(self.data)):
+            matrix_in_lines += (str(self.data[i]) + "\n")
+        return matrix_in_lines
+
+    @staticmethod
+    def unity(size):
+        """ unity method creates a matrix of zeros with a diagonal of ones
```

Comment:
give more attention to how you write your documentation.
notice:
- **Capital** letter at the start of the sentence
- Dot at the end of each sentence `.`